src/connect.ts

```diff
@@ -29,12 +29,17 @@ export type MutationMapper =
   | string
   | ((this: Vue, commit: Commit, ...args: any[]) => any)
 
+export type StateMapperFunction = (mapper: typeof mapState) => Record<string, () => any>;
+export type GetterMapperFunction = (mapper: typeof mapGetters) => Record<string, () => any>; (propName: string) => Function;
+export type ActionMapperFunction = (mapper: typeof mapActions) => Record<string, () => any>; (propName: string) => Function;
```

Comment:
`ActionMapperFunction` and `MutationMapperFunction` should be

```ts
(mapper: typeof mapActions) => Record<string, Function>
(mapper: typeof mapMutations) => Record<string, Function>
```

...because they will return `methods` rather than `computed`.